public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -592,9 +592,10 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
     e.stopPropagation();
     locationService.partial({ inspect: this.props.panel.id, inspectTab: tab });
   };
-  onOpenErrorInspect(e: React.SyntheticEvent, tab: string) {
+
+  onOpenErrorInspect(e: React.SyntheticEvent) {
```

Comment:
pushed a small fix here, `this` is undefined unless we bind the method or use arrow function

Reply:
@polibb ah, good catch, since it was prefixed with onXXX I just assumed this was an arrow function :)